.storybook/preview.js

```diff
@@ -44,9 +44,7 @@ const withThemeProvider = (Story, context) => {
   return (
     <ThemeProvider theme={{ ...DEFAULT_THEME, rtl }}>
       <GlobalPreviewStyling />
-      <StyledExampleWrapper>
-        <Story />
-      </StyledExampleWrapper>
+      <StyledExampleWrapper>{Story()}</StyledExampleWrapper>
```

Comment:
What is this necessary for? I thought these were equivalent?

Reply:
It's a work-around to fix transition issues discussed in https://github.com/zendeskgarden/react-components/pull/889#discussion_r496912845. There's an open issue in Storybook and this is the work-around: https://github.com/storybookjs/storybook/issues/12255

Reply:
Added an inline comment that clarifies why this change is needed.